waku/waku_archive/archive.nim

```diff
@@ -55,9 +55,6 @@ proc validate*(msg: WakuMessage): Result[void, string] =
     # Ephemeral message, do not store
     return
 
-  if msg.timestamp == 0:
-    return ok()
-
```

Comment:
I agree that we shouldn't have this validation because the WakuMessage spec doesn't enforce it. https://rfc.vac.dev/waku/standards/core/14/message
Let me know if that statement is correct

Reply:
This the the spec section, for reference -> https://github.com/vacp2p/rfc/tree/docs/new-store-v3-protocol/content/docs/rfcs/13#waku-message-store-eligibility